aspnetcore/grpc/loadbalancing.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Scalable apps with gRPC client-side load balancing
+author: jamesnk
+description: Learn how to make scalable, high-performance gRPC apps with client-side load balancing in .NET.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: jamesnk
+ms.date: 05/18/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: grpc/loadbalancing
+---
+# Scalable apps with gRPC client-side load balancing
+
+By [James Newton-King](https://twitter.com/jamesnk)
+
+gRPC client-side load balancing is a feature that allows gRPC clients to distribute load optimally across available servers. This article discusses how to configure a client-side load balancing to create scalable, high-performance gRPC apps in .NET.
+
+gRPC client-side load balancing requires [Grpc.Net.Client](https://www.nuget.org/packages/Grpc.Net.Client) version XXXX or later.
+
+## Why load balancing is important
+
+HTTP/2 multiplexes multiple calls on a single TCP connection. If gRPC and HTTP/2 are used with a network load balancer, the connection is forwarded to a server and then all gRPC calls are sent to that one server. The other server instances the network load balancer is supposed to distribute load to are idle.
+
+Network load balancers are a common solution for load balancing because they are fast and lightweight. For example, Kubernetes will use a network load balancer to balance connections between pod instances by default. However, network load balancers are not effective at distributing load when used with gRPC and HTTP/2.
```

Comment:
```suggestion
Network load balancers are a common solution for load balancing because they are fast and lightweight. For example, Kubernetes by default uses  a network load balancer to balance connections between pod instances. However, network load balancers are not effective at distributing load when used with gRPC and HTTP/2.
```